### DISCUSSION TITLE
Multiple Different Dark Themes?

### DISCUSSION THREAD

**Discussion by MrDogeBro** [MrDogeBro/sphinx_rtd_dark_mode#8] [Ideas]
@KrishnanSG pointed out that some users might thing the dark theme is too dark. I personally like a dark dark theme but I suggested the idea of having multiple different dark themes. I would like to hear what people think about this idea. If people would like it, it would also be helpful to know what general color schemes people would like in the themes as well.

**Comment by MrDogeBro** [comment]
There could be a selector on the site to the let the viewer choose their theme and there would also be a config option for the default theme to display to the user on their first load. How does that all sound?

**Reply by KrishnanSG** [reply]
@MrDogeBro I feel the selection should be done during the build and not as an selector on the site.

The dark theme I really liked was `slate` used by [material theme for mkdocs](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/)

Ex:
![image](https://user-images.githubusercontent.com/43802499/118589474-180b6000-b7be-11eb-91ea-0ed646e1d65b.png)

**Reply by MrDogeBro** [reply]
@KrishnanSG currently on the site there is a button to toggle between light and dark mode. It would be a similar idea, just with options instead of a toggle button if that makes sense.

For example, it would be very similar to how something like the Rust docs do it (see screenshot below). The person making the docs would have the option of which theme is the default, but the user would still have their choice.

Also, this is kind of like a more long term idea, but there could potentially be a system put in place that would allow for users to customize the themes or even create their own themes. As a part of this, we could also have a way of publishing those themes as community ones potentially so that others could install them and use the theme that person created. That's just an idea I'm throwing out there so feedback on that would be great!

![Cap 05-19-21 16 01 29](https://user-images.githubusercontent.com/35872177/118876649-7ff3a080-b8bb-11eb-8d12-6159127f24b4.png)

**Reply by KrishnanSG** [reply]
@MrDogeBro I love this idea, looking forward to use it.

**Reply by MrDogeBro** [reply]
@KrishnanSG ok so I have added a new theme (slate) which is based off of the one you sent above. It will come out as a part of the v2 release.

**Reply by KrishnanSG** [reply]
Wow! Interesting
Btw when would v2 be released?

**Reply by MrDogeBro** [reply]
Not quite sure yet. I am currently working on the v2 release. When it gets closer to completion I might add a date to the milestone which would be when I hope to release it.

**Reply by kassbohm** [reply]
I also think, it is a great idea to let the viewer choose between different (dark and less dark) themes... But now that I read the date... are you still working on this, MrDogeBro?

**Comment by Zethson** [comment]
@MrDogeBro love it. More choices are always better if you can support them well.
